apstools/utils/device_info.py

```diff
@@ -48,7 +50,7 @@ def _all_signals(base):
     return items
 
 
-def _get_named_child(obj, nm):
+def _get_named_child(obj: Device, nm: str) -> Optional[Union[Device, str]]:
```

Comment:
Not sure how this works but either OphydObject or None are returned:

```suggestion
def _get_named_child(obj: Device, nm: str) -> Union[OphydObject, NoneType]:
```

Reply:
object could be a Device or a Signal